src/rpc/output_script.cpp

```diff
@@ -222,12 +222,14 @@ static RPCHelpMan deriveaddresses()
     return RPCHelpMan{"deriveaddresses",
         {"\nDerives one or more addresses corresponding to an output descriptor.\n"
          "Examples of output descriptors are:\n"
-         "    pkh(<pubkey>)                        P2PKH outputs for the given pubkey\n"
-         "    wpkh(<pubkey>)                       Native segwit P2PKH outputs for the given pubkey\n"
-         "    sh(multi(<n>,<pubkey>,<pubkey>,...)) P2SH-multisig outputs for the given threshold and pubkeys\n"
-         "    raw(<hex script>)                    Outputs whose scriptPubKey equals the specified hex scripts\n"
+         "    pkh(<pubkey>)                                 P2PKH outputs for the given pubkey\n"
+         "    wpkh(<pubkey>)                                Native segwit P2PKH outputs for the given pubkey\n"
+         "    sh(multi(<n>,<pubkey>,<pubkey>,...))          P2SH-multisig outputs for the given threshold and pubkeys\n"
+         "    raw(<hex script>)                             Outputs whose scriptPubKey equals the specified hex scripts\n"
+         "    tr(KI,multi_a(<n>,<pubkey>,<pubkey>,...))     P2TR-multisig outputs for the given threshold and pubkeys\n"
```

Comment:
why not <pubkey> instead of KI? I think it's more confusing trying to explain everything vs giving a hint then pointing the user to the doc.

Reply:
Thanks! I'm not clear on your question though.

When you say "why not instead of KI"? Instead of what?

Reply:
oh github ate the term: `<pubkey>`

Reply:
I agree this would be cleaner. I will make the change.

Reply:
Thanks, I've updated the commit to address your review comments. Most appreciated.